56.md

```diff
@@ -48,35 +53,79 @@ Example events
 {
   "kind": 1984,
   "tags": [
-    [ "e", <eventId>, "illegal"],
-    [ "p", <pubkey>]
+    [ "p", <pubkey>, "nsfw", "", "-"],
+    [ "p", <pubkey>, "sfw"]
+  ],
+  "content": "It's a cute dog!",
+  ...
+}
+
+{
+  "kind": 1984,
+  "tags": [
+    ["e", <event id>, "spam"],
+    ["p", <pubkey>]
   ],
-  "content": "He's insulting the king!",
+  "content": "Spam post",
   ...
 }
 
 {
   "kind": 1984,
   "tags": [
-    [ "p", <impersonator pubkey>, "impersonation"],
-    [ "p", <victim pubkey>]
+    ["e", <event id>, "spam", "topic"],
+    ["p", <pubkey>]
   ],
-  "content": "Profile is imitating #[1]",
+  "content": "Discusses spam",
   ...
 }
 ```
 
 Client behavior
 ---------------
 
-Clients can use reports from friends to make moderation decisions if they
-choose to. For instance, if 3+ of your friends report a profile as explicit,
-clients can have an option to automatically blur photos from said account.
+Labels can be used the same way as hashtags, they can be used for filtering,
+searching, or displayed to draw attention to something.
 
+Clients should have a threshold for labels to be considered legitimate,
```

Comment:
I'd change "should" to "may"

Reply:
You're right. I wanted to indicate that this should be done somehow, something analagous to the previous "It is not recommended that relays perform automatic moderation using reports." But that's true of all events in Nostr and it may be alright to assume the reader already knows.

Reply:
Yeah I agree that it should be done in most cases, but something doesn't feel right about wording it that way in the spec